frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditMobile/SmsCountsModal.tsx

```diff
@@ -0,0 +1,82 @@
+import { useParams } from 'react-router-dom'
+import {
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from '@chakra-ui/react'
+
+import { SmsCountsDto } from '~shared/types/form'
+
+import { ADMINFORM_ROUTE, ADMINFORM_SETTINGS_SUBROUTE } from '~constants/routes'
+import { useIsMobile } from '~hooks/useIsMobile'
+import Badge from '~components/Badge'
+import Button from '~components/Button'
+import Link from '~components/Link'
+import { ModalCloseButton } from '~components/Modal'
+import Spinner from '~components/Spinner'
+
+import { formatSmsCounts } from './utils'
+
+type SmsCountsModalProps = {
+  freeSmsCount?: SmsCountsDto
+  isOpen: boolean
+  onClose: () => void
+}
+export const SmsCountsModal = ({
+  freeSmsCount,
+  isOpen,
+  onClose,
+}: SmsCountsModalProps) => {
+  const { formId } = useParams()
+  const isMobile = useIsMobile()
+
+  return (
+    <Modal isOpen={isOpen} onClose={onClose}>
+      <ModalOverlay />
+      <ModalContent>
+        {isMobile ? null : <ModalCloseButton />}
+        <ModalHeader>Verified SMS Billing</ModalHeader>
+        <ModalBody>
+          {freeSmsCount === undefined ? (
+            <Spinner fontSize="2rem" />
+          ) : (
+            <>
+              <Text textStyle="body-2">
+                Form provides {`${freeSmsCount.quota.toLocaleString()}`} free
+                SMS OTP verifications per account, for forms you are an owner
+                of. Once this limit is reached, SMS OTP verification will be
+                automatically disabled for all owned forms. Forms with Twilio
+                already set up will not be affected.
+              </Text>
+              <Text textStyle="body-2" mt="1.5rem">
+                If you are a collaborator, ensure the form's owner has enough
+                free verifications. If you require more than{' '}
+                {`${freeSmsCount.quota.toLocaleString()}`} verifications, please{' '}
```

Comment:
Why are the `{' '}` needed?

Reply:
extra space, or formatter will remove the space